lib/FCM/Util/ConfigReader.pm

```diff
@@ -310,7 +310,6 @@ sub _parse_fcm2_var {
                 $E->CONFIG_VAR_UNDEF, $state_ref->{ctx}, "undef($symbol)",
             );
         }
-        $substitute ||= q{};
```

Comment:
This line is unnecessary. The above block (starting at line 308) should prevent `undef` value.